crates/ra_syntax/src/syntax_error.rs

```diff
@@ -111,6 +120,44 @@ impl fmt::Display for SyntaxErrorKind {
     }
 }
 
+impl fmt::Display for TokenizeError {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        let msg = match self {
+            TokenizeError::EmptyInt => "Missing digits after integer base prefix",
+            TokenizeError::EmptyExponent => "Missing digits after the exponent symbol",
```

Comment:
I wish `rustfmt` was not so clippy, initialy I wrote this match with each arm homogeneously and nicely wrapped into curly braces, but the formatting tool demands the shorthand syntax here ...

Reply:
You could always `#[rustfmt::skip]` this one function. We do the same for a few match statements elsewhere.